src/reducers/l10n.js

```diff
@@ -3,18 +3,22 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 // @flow
-import type { Reducer, L10nState } from 'firefox-profiler/types';
+import type {
+  Reducer,
+  L10nState,
+  L10nFetchingPhase,
+} from 'firefox-profiler/types';
 import { ReactLocalization } from '@fluent/react';
 import { combineReducers } from 'redux';
 
-const isL10nFetching: Reducer<boolean> = (state = false, action) => {
+const isL10nFetching: Reducer<L10nFetchingPhase> = (state, action) => {
```

Comment:
It's more idiomatic for our codebase to add a default value to the `state` and return the state instead.
So it here should be `state = 'not-fetching'` and default of the switch statement should directly return `state`.

Also, as I mentioned my other comment, it would be good to rename this to something like `l10nFetchingPhase` as well.